modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js

```diff
@@ -97,6 +99,69 @@ const useFormSubmit = ({apiRef, containerRef}) => {
 		[apiRef, containerRef, submittable]
 	);
 
+	const handleFormSubmitted = useCallback(
+		(event) => {
+			event.preventDefault();
+
+			if (portletNamespace === DDM_FORM_PORTLET_NAMESPACE) {
```

Comment:
@mlan0, can you please send a new complementary pull replacing this code by this [code](https://github.com/natocesarrego/liferay-portal/pull/1100/commits/08c695ab9afee269c013d57e3914721c037ce384)?

This modification is necessary to allow this fix work on a Form Portlet added to a Site Page as a Widget. If you have any question, please let me know.

Thanks

Reply:
Done!